src/wai_data_tools/scripts/cli.py

```diff
@@ -28,6 +28,12 @@ def cli() -> None:
     help="Path to the excel file with label information",
     required=True,
 )
+@click.option(
+    "--config_filepath",
+    type=click.Path(exists=True, path_type=pathlib.Path),
+    help="Path to the configuration file",
+    required=True,
```

Comment:
can't the user just user the defaults?

Reply:
They can but I am a bit unsure how to set it up so I can read the default config when I build the package since I need some form of relative path from the run file to the default config. I will see if I can get something to work.

Reply:
err, they should* currently they need to give the path to a config file

Reply:
I pushed a proposal to solve this. In summary I include a default config using a MANIFESTO.in file to include the .yml file when building and then I construct a relative path from the entrypoint file to the default config file at the top of the cli.py module

Reply:
We should consider on how the usuer should interact with this package and it seems unlikely that a user will start using manifestos in my opinion

Reply:
I am a bit confused. The user wouldn't use the manifesto file. It is just used to include the default config in the package. If the user want to use their own config they just need to specify the path when calling a command